### ISSUE TITLE
Fix broken images in the contribution.md file of this repo

### ISSUE THREAD

**Issue by TimOsahenru** [BlackPythonDevs/.github#18]
You can create an `assets/images` folder and add the relevant files using this [**REPO**](https://github.com/BlackPythonDevs/blackpythondevs.github.io) as a guide or source of truth on what files to include in the `assets/images`

**Comment by mbvgua** [comment]
Yes images would better help one visualize this. 

But a follow up question that adds on that: since this specific repo only has markdown files(as compared to the github.io repo that needs alot of dependecies and packages installed), wouldnt it be easier and better if people were to work on them in their local environment using their favourite editor instead of the Codespaces?

**Comment by TimOsahenru** [comment]
Not sure what you mean, do you mind clarifying? Mind you, this issue is to fix the broken images in this file https://github.com/BlackPythonDevs/.github/blob/main/CONTRIBUTING.md

**Comment by mbvgua** [comment]
Okay, great. Your link directs to the page below:

![Screenshot from 2024-09-16 18-58-24](https://github.com/user-attachments/assets/9e2d418e-3198-4c72-b1d3-a79b49298310)

And I think that you are proposing for the links to be fixed such that they will resemble the main repo as below:

![Screenshot from 2024-09-16 19-09-08](https://github.com/user-attachments/assets/45d4e7b8-95aa-4dc4-9263-f3513ead2dee)

Your proposed solution would aim for the images to direct one on how to develop on the Github Codespaces. I am building on your idea by proposing that for this  specific repo, instead of working in the codespaces we could instead work in a local environment, whereby after forking the repo you clone it to your device then begin contributing.

If these were the case, the images that would need to be uploaded would not have to for codespaces as you suggested, but instead ones guiding you on how to develop locally.

**Comment by TimOsahenru** [comment]
Gotcha! Well this issue is to help those contributing using the codespace but, can’t make meaningful sense out of it cos of the broken images

**Comment by TimOsahenru** [comment]
If there isn’t an issue that deals with contributing locally I’d suggest you create one

**Comment by mbvgua** [comment]
Yes. I also agree that would be better way forward. Let me do that immediately

**Comment by TimOsahenru** [comment]
Ensure you confirm that issue has not already been created. Check this out https://github.com/BlackPythonDevs/blackpythondevs.github.io/pull/416 might be related to your issue

**Comment by mbvgua** [comment]
I think that the organisations repos are supposed to be independent of each other. Because if this were the case, the issue has not been raised on this repository, but if not I have seen it in another repo [here](https://github.com/BlackPythonDevs/blackpythondevs.github.io/issues/401) .

Edit: after thorough consideration, I think I will just close the issue i had created. If it is fixed on the main repo the changes might reflect here too.

If not, ill be sure to open it again :smile:
